Homework7/MobilePhone.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Homework7
+{
+    public class MobilePhone : Item, IPhoto
+    {
+        public double numberOfPixelsInCamera;
+        public double NumberOfPixelsInCamera { get { return 0; } set { numberOfPixelsInCamera = value; } }
```

Comment:
Why does it returns 0? The same in Polaroid class

Reply:
fixed to return numberOfPixelsInCamera